core/package.json

```diff
@@ -126,6 +128,7 @@
 		"remark-rehype": "^11.1.0",
 		"remark-stringify": "^11.0.0",
 		"remove": "^0.1.5",
+		"require-in-the-middle": "^7.4.0",
```

Comment:
i remember this and `import-in-the-middle` being necessary due to how `serverExternalPackages` setting in `next.config.js` now works. it just threw some warnings, but this got rid of it.

ideally experiment with removing this